setup.py

```diff
@@ -24,6 +24,12 @@
 with open(os.path.join(ROOT, 'README.md'), encoding="utf-8") as f:
     README = f.read()
 
+# add platform dependant optional compilation argument
+opt_arg=["-O3"]
+import platform
+if platform.system()=='Darwin':
+  if platform.release()=='18.0.0':
```

Comment:
shouldn't you use >= instead, i don't think this option will be necessary on just of release of MacSOX

```suggestion
  if platform.release()>='18.0.0':
```

PS trying the suggestion thing it's nice.

Reply:
not sure about the suggestion. In its form I think the bug is partly due to the fact that python Extension module has trouble to get the correct path from the new locations of clang compiler on the new OSX. I hope this will be corrected in the future. Clearly it is a temporary patch here.